File: Week 3/C++/PALL01.cpp
#include <iostream>
using namespace std;

int main() {
	int t;
	
	cin >> t;
	for (int i = 0; i < t; ++i) {
	    int  n, temp, rev = 0;
	    cin >> n;
	    temp = n;
	    while (temp) {
	        rev = rev * 10 + temp % 10;
	        temp /= 10;
	    }
	    if (n == rev)
	        cout << "wins" << endl;
	    else
	        cout << "loses" << endl;
	}
	    
	return 0;
}

File: Week 3/Java/PALL01.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Codechef
{
	static int reverse(int num) {
        int rev = 0;
        while(num > 0) {
            int d = num % 10;
            rev = rev * 10 + d;
            num = num / 10;
        }
        return rev;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        int ar[];
        ar = new int[t];
        for(int i = 0; i < t; i++) {
            ar[i] = sc.nextInt();
        }
        for(int j = 0; j < t; j++) {
            if(ar[j] == reverse(ar[j])) {
                System.out.println("wins");
            } else {
                System.out.println("loses");
            }
        }
    }
}

